pkg/daemon/rpm-ostree.go

```diff
@@ -303,3 +348,23 @@ func runGetOut(command string, args ...string) ([]byte, error) {
 	}
 	return rawOut, nil
 }
+
+func (state *rpmOstreeState) getBootedDeployment() (*RpmOstreeDeployment, error) {
```

Comment:
(Tangentially related to this I was kind of thinking about splitting out a github.com/coreos/rpmostree-client-go repository)